### ISSUE TITLE
é and è accents are inverted

### ISSUE THREAD

**Issue by ferenax** [itfoundry/laila#1]
Hello, 

I've been working with your font when building a new website for a french client. However, I noticed that é and è accents were inverted in the current version available on google fonts website.

Could you please fix this issue and reupload this font, which is very beautiful and that I'd rather keep on the website.

Thank you,

**Comment by ferenax** [comment]
I guess this issue was fixed in here, but is still showing in Google Fonts, could you please reupload it to Google Fonts ? Thank you

**Comment by pelemele01** [comment]
Hello,
exactly the same problem : é, è à, ù accents were inverted on google font.

Thank you.

**Comment by pacproduct** [comment]
I can confirm this issue still exists in 2018. Could that be fixed?

**Comment by davelab6** [comment]
@m4rc1e please prioritize this :)

**Comment by m4rc1e** [comment]
Yep, this is fixed in the latest release. However, there are some missing glyphs, #2

**Comment by pacproduct** [comment]
Yes indeed, I can confirm that `é`, `è` and `à` work fine in the latest release, *here on GitHub*.
However the font downloadable from the fonts.google.com website is still the wrong one.
I'm not familiar with how the Google platform works but... surely there's a way to refresh it / update it?

**Comment by m4rc1e** [comment]
@pacproduct well the new version hosted here is missing some glyphs, https://github.com/itfoundry/laila/issues/2. I can't update the GF release until these glyphs are added or deemed unnecessary.

**Comment by pacproduct** [comment]
@m4rc1e : Fair enough :)
Forgot to mention that it's otherwise an amazing Font that I find very classy.

**Comment by LucMorizur** [comment]
Hi;
in fact it's all cute and grave accents which are swapped: the issue is the same for à,ù,ì...
Success!

**Comment by pantic4114** [comment]
> Hi;
> in fact it's all cute and grave accents which are swapped: the issue is the same for à,ù,ì...
> Success!



> Yes indeed, I can confirm that `é`, `è` and `à` work fine in the latest release, _here on GitHub_.
> However the font downloadable from the fonts.google.com website is still the wrong one.
> I'm not familiar with how the Google platform works but... surely there's a way to refresh it / update it?

Where i can found the font with accents resolved?

**Comment by oitent** [comment]
> Hello,
> 
> I've been working with your font when building a new website for a french client. However, I noticed that é and è accents were inverted in the current version available on google fonts website.
> 
> Could you please fix this issue and reupload this font, which is very beautiful and that I'd rather keep on the website.
> 
> Thank you,

I have the exact same problem!
![image](https://user-images.githubusercontent.com/27972873/57861401-e9241300-77cc-11e9-93c6-ccdfeaca6235.png)
![image](https://user-images.githubusercontent.com/27972873/57861427-f4773e80-77cc-11e9-83c4-00547379553d.png)

fonts.google.com still not updated.

**Comment by bybpow** [comment]
18-11-2020 
This problem still exists in Google Fonts.

**Comment by jerobarraco** [comment]
08-2021
This problem still exists in Google Fonts.

**Comment by PatrickJunod** [comment]
07-2022
This problem still exists in Google Fonts.

**Comment by chrissimpkins** [comment]
Have the issues been addressed that previously prevented this from being onboarded in Google Fonts? We continue to receive user reports about this issue but it isn't clear from this thread if there are outstanding issues to address before we can onboard the fixes.

**Comment by Cheese-Slice-1** [comment]
Yep, still exists. Please, someone, fix this!

**Comment by rokapoz** [comment]
@chrissimpkins Do you know what the status of the fixes are for this? fonts.google.com is still showing the accents inverted. 
![image](https://github.com/itfoundry/laila/assets/46518820/5aa809d6-862e-4a6b-87cc-f270d9e4df7a)

**Comment by chrissimpkins** [comment]
I'll add it to the Q3 queue so that we can investigate. https://github.com/itfoundry/laila/issues/1#issuecomment-374570621 has been the blocker in the past as I understand the discussion in this thread.  Though we should be able to roll out a fix in a previously accepted character set without a requirement to expand the glyph repertoire.  

Will discuss, have a look at where the project is at the moment, and decide how to approach it over the upcomign quarter.
